curriculum/challenges/english/25-front-end-development/lab-rpg-character/67d83df6f82eda3868dd2a84.md

```diff
@@ -0,0 +1,227 @@
+---
+id: 67d83df6f82eda3868dd2a84
+title: Build an RPG Character
+challengeType: 27
+dashedName: build-an-rpg-character
+---
+
+# --description--
+
+Fulfill the user stories below and get all the tests to pass to complete the lab.
```

Comment:
Also, I am starting to review another python lab and that one has a couple of sentence explaining what the project is about. So we should do that here for consistency but also to provide extra context for campers on what they are going to build.